.travis.yml

```diff
@@ -44,6 +44,12 @@ jobs:
         - BUILD=unit
         - BUILD_TYPE=Release
         - CXXFLAGS="-std=c++17 -fconcepts"
+    - name: "Unit gcc10 (c++2a)"
```

Comment:
```suggestion
    - name: "Unit gcc10 (c++20)"
```
?

Reply:
It is technically only C++2a, because the standard wasn't finalised, yet.

Reply:
Ok, was wondering, because you can pass `-std=c++20` to GCC10

Reply:
Okay, if it is like that we use -std=c++20 instead :)